pages/k8s/charm-kubernetes-worker.md

```diff
@@ -0,0 +1,355 @@
+---
+wrapper_template: "kubernetes/docs/base_docs.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "kubernetes-worker charm"
+  description: Etcd Charm reference
+keywords: kubernetes-worker, charm, config
+tags: [reference]
+sidebar: k8smain-sidebar
+permalink: charm-kubernetes-worker.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+This charm deploys a container runtime, and additionally stands up the Kubernetes
+worker applications: kubelet, and kube-proxy.
+
+In order for this charm to be useful, it should be deployed with its companion
+charm kubernetes-master and linked with an SDN-Plugin and a container runtime
+such as containerd.
+
+This charm has also been bundled up for your convenience so you can skip the
+above steps, and deploy it with a single command:
+
+```bash
+juju deploy charmed-kubernetes
+```
+
+For more information about Charmed Kubernetes see the [overview documentation](/kubernetes/docs/overview)
+
+## Scale out
+
+To add additional compute capacity to your Kubernetes workers, you may
+`juju add-unit` scale the cluster of applications. They will automatically
+join any related kubernetes-master, and enlist themselves as ready once the
+deployment is complete.
+
+## Snap Configuration
+
+The Kubernetes resources used by this charm are snap packages. When not
+specified during deployment, these resources come from the public store. By
+default, the snapd daemon will refresh all snaps installed from the store
+four (4) times per day. A charm configuration option is provided for operators
+to control this refresh frequency.
+
+NOTE: this is a global configuration option and will affect the refresh
+time for all snaps installed on a system.
+
+### Examples:
+
+##### refresh kubernetes-worker snaps every tuesday
+
+```bash
+juju config kubernetes-worker snapd_refresh="tue"
+```
+
+##### refresh snaps at 11pm on the last (5th) friday of the month
+
+```bash
+juju config kubernetes-worker snapd_refresh="fri5,23:00"
+```
+##### delay the refresh as long as possible
+
+```bash
+juju config kubernetes-worker snapd_refresh="max"
+```
+
+##### use the system default refresh timer
+
+```bash
+juju config kubernetes-worker snapd_refresh=""
+```
+
+For more information on the possible values for snapd_refresh, see the
+refresh.timer section in the system options documentation.
```

Comment:
As with the k8s-master charm, let's change this to something like: "For more information, see the /kubernetes/docs/snap-refresh documentation."